torch/nn/modules/rnn.py

```diff
@@ -368,6 +369,7 @@ class LSTM(RNNBase):
           :func:`torch.nn.utils.rnn.pack_sequence` for details.
         - **h_0** of shape `(num_layers * num_directions, batch, hidden_size)`: tensor
           containing the initial hidden state for each element in the batch.
+          If the RNN is bidirectional, num_directions should be 2, else it should be 1.
```

Comment:
I don't really like how this is repeated four times, is it possible to add it above the "Inputs: input, h_0" line?

Something like:
```
For the purposes of computing input/output tensor sizes, num_directions equals 2 if the RNN is bidirectional, otherwise 1.
```

Reply:
Sure, but I added the docs for the `RNN`, `GRU`, and `LSTM` class. If we don't want to repeat it, then should it only be added for one class (e.g. RNN?) Or should we keep it for all 3?

Reply:
Sorry, I didn't realize that many of the repeats were for adding the note to RNN, GRU, and LSTM. Let's keep those repeats, but only mention num_directions once per RNN, GRU, and LSTM docstring